release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3116,6 +3116,66 @@ $ ccoctl azure create-all \
 --
 (link:https://issues.redhat.com/browse/OCPBUGS-22651[*OCPBUGS-22651*])
 
+* When installing an {product-title} cluster with static IP addressing or dual-stack networking with Tang encryption, worker nodes boot without network settings. This condition prevents worker nodes from accessing the Tang server, causing installation to fail. To address this condition, you must set the network settings as kernel arguments.
++
+When using installer-provisioned installation, provide kernel arguments using a manifest. For example: 
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  name: 99-worker-tang-network-config
+  labels:
+    machineconfiguration.openshift.io/role: worker
+spec:
+  config:
+    ignition:
+      version: 3.1.0
+  kernelArguments:
+    - 'ip=<static-ip>::<gateway>:<netmask>:<hostname>:<interface>:none'
+    - 'rd.neednet=1'
+----
++
+Save and copy the manifest file to the `clusterconfigs/openshift` directory:
++
+[source,terminal]
+----
+$ cp ~/router-replicas.yaml clusterconfigs/openshift/99-worker-tang-network-config.yaml
+----
++
+Apply the manifest: 
++
+[source,terminal]
+----
+$ oc apply -f clusterconfigs/openshift/99-worker-tang-network-config.yaml
+----
++
+When installing with {ai-full}, modify the infrastructure environment using the API to include the network settings as kernel arguments. For example:
++
+[source,terminal]
+----
+$ curl https://api.openshift.com/api/assisted-install/v2/infra-envs/${INFRA_ENV_ID} \
+-X PATCH \
+-H "Authorization: Bearer ${API_TOKEN}" \
+-H "Content-Type: application/json" \
+-d "$(jq --null-input \
+  --slurpfile pull_secret ~/Downloads/pull-secret.txt '
+    {
+      "kernel_arguments": [
```

Comment:
kernel arguments are per host, so we used https://github.com/openshift/assisted-service/blob/b3565e7373652cf8a7ede6d8d1d04583e019994b/api/v1beta1/agent_types.go#L192 instead of infra env variables

Reply:
@filanov Ok, I've removed the dual stack reference and done some additional editing for clarity.